excel/client.bal

```diff
@@ -257,9 +258,34 @@ public client class Client {
         return getRowArray(response);
     }
 
+    # Updates the properties of a row.
+    #
+    # + workbookIdOrPath - Workbook ID or file path. Path should be with the `.xlsx` extension from root. If a workbook
+    # is in root, path will be `<FILE_NAME>.xlsx`)
+    # + worksheetNameOrId - Worksheet name or ID
+    # + tableNameOrId - Table name or ID
+    # + index - The index number of the row within the rows collection of the table
+    # + values - A 2-dimensional array of unformatted values of the table rows (boolean or string or number). Provide
+    # values for relevant fields that should be updated. Existing properties that are not included in the request will
+    # maintain their previous values or be recalculated based on changes to other property values. For best performance
+    # you shouldn't include existing values that haven't changed
+    # + return - `Row` record or error
+    @display {label: "Update Row"}
+    remote isolated function updateRow(@display {label: "Workbook ID or Path"} string workbookIdOrPath, 
+                                        @display {label: "Worksheet Name or ID"} string worksheetNameOrId, 
+                                        @display {label: "Table Name or ID"} string tableNameOrId,
+                                        @display {label: "Index"} int index,
+                                        @display {label: "Values"} json[][] values)
+                                        returns Row|error {
+        string path = check createRequestPath([WORKSHEETS, worksheetNameOrId, TABLES, tableNameOrId, ROWS,  ITEM_AT + 
+        OPEN_ROUND_BRACKET + INDEX + EQUAL_SIGN + index.toString() + CLOSE_ROUND_BRACKET], workbookIdOrPath);
```

Comment:
Shall we fix the code formatting issue?

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerinax-microsoft.excel/commit/18d6f684920ab5ffac6845fc79db985b185cbbb3